runtime/src/bank.rs

```diff
@@ -1011,6 +1016,87 @@ pub struct ProcessedTransactionCounts {
     pub signature_count: u64,
 }
 
+/// Account stats for computing the bank hash
+/// This struct is serialized and stored in the snapshot.
+#[cfg_attr(feature = "frozen-abi", derive(AbiExample))]
+#[derive(Clone, Default, Debug, Serialize, Deserialize, PartialEq, Eq)]
+pub struct BankHashStats {
+    pub num_updated_accounts: u64,
+    pub num_removed_accounts: u64,
+    pub num_lamports_stored: u64,
+    pub total_data_len: u64,
+    pub num_executable_accounts: u64,
```

Comment:
i'm struggling to see where these are reported. What am I missing?

Reply:
it is not reported in metric. We logged it when we freeze the bank.

https://github.com/anza-xyz/agave/blob/df27fb3a7386f0d2cc64df81c46a09ab79b18ba0/runtime/src/bank.rs#L5644

seems to only used for debugging consensus issue.

Reply:
ah. I was looking for a use of the specific fields. I didn't realize the entire thing was debug logged.